config/cscs.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016-2023 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# Copyright 2016-2024 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
```

Comment:
what do you think of https://hynek.me/til/copyright-years/ ?

Reply:
Good point, we can stop updating 👍 I will change this